coreneuron/utils/progressbar/progressbar.c

```diff
@@ -32,6 +32,18 @@ enum { WHITESPACE_LENGTH = 2 };
 /// The amount of width taken up by the border of the bar component.
 enum { BAR_BORDER_WIDTH = 2 };
 
+/// Progress Bar max number of updates
+enum { BAR_UPDATES_NUMBER = 500 };
+
+/// Progress Bar interval of updates (seconds)
+enum { BAR_UPDATES_INTERVAL = 5 };
+
+/// Progress Bar time of change of updates (seconds)
+enum { BAR_UPDATES_CHANGE = 120 };
+
+/// Progress Bar number of slow updates
+enum { BAR_UPDATES_SLOW = BAR_UPDATES_NUMBER - BAR_UPDATES_INTERVAL * BAR_UPDATES_CHANGE };
```

Comment:
I'm not getting the logic, sorry, maybe tomorrow you can explain me. But this is resulting in a constant -100.